src/globus_cli/commands/flows/create.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import annotations
+
+import typing as t
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import JSONStringOrFile, command
+from globus_cli.termio import Field, TextMode, display, formatters
+
+ROLE_TYPES = ("flow_viewer", "flow_starter", "flow_administrator", "flow_owner")
+
+
+@command("create", short_help="Create a flow")
+@click.argument(
+    "title",
+    type=str,
+)
+@click.argument(
+    "definition",
+    type=JSONStringOrFile(),
```

Comment:
The more I think about this, the less I like the requirement to specify `file:` as a prefix to the filename. I'd prefer to lean away from that syntax and instead assume that if the file exists, it's a file; and if it doesn't exist, fall back to loading the text as JSON.

I'm also leery of tab completion getting broken by mandating a `file:` prefix.

Reply:
With time, I've also started to wonder if we can redo the json-or-file parsing. I originally introduced it in gcs-cli because we had other things which needed a similar parsing strategy, and then I carried it back to globus-cli to keep things consistent.

I'd like to talk about this today and see if we can write something like... `JSONStringOrFileV2` which has behavior more to our liking.

Reply:
I'm hoping that I have this resolved in a way which we will like in #790.
I'd like us to hold this PR until that can get review, as the resolution here may be (:crossed_fingers:) a simple rebase.